db/app.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const app = express();
+const {
+    getCategories
+} = require('../db/controllers/categoryControllers');
+const { 
+    handle500Errors, 
+    handleCustomErrors,
+    handle400Errors 
+} = require('./controllers/error-handling-controllers')
+
+app.get('/api/categories', getCategories)
+
+app.use(handle400Errors)
```

Comment:
you have added a lot of unncessesary boiler plate code here, you do not need to have any of these error handlers in here other than the 500, please be careful to only write the code that you need to pass the tests that you have written